the-then-years-bug/the-ten-years-bug.md

```diff
@@ -19,11 +19,11 @@ We simply create a new script tag on the hosting page, which loads our main reco
 
 First symptoms
 ------------
-It all started with a collection of unrelated bugs. An 'unable to get property '2' of undefined' here, an 'undefined is not a function' there. All seemingly minor bugs. But for some reason those became legendary. We fixed them, but they persisted. We could not longer recreate them, but they kept popping up in the logs consistently and regularly. Many hours were spent in the attempt to debug them.
+It all started with a collection of unrelated bugs. An 'unable to get property '2' of undefined' here, an 'undefined is not a function' there. All seemingly minor bugs. But in some reason those became legendary. We fixed them, but they persisted. We could not recreate them, but they kept popping up in the logs consistently and regularly. Many hours were spent in the attempt to debug them.
```

Comment:
Why in? pretty sure it's "for some reason"

I think the "longer" should stay there. But it should be "no longer" instead of "not longer"